### DISCUSSION TITLE
why does vllm 0.6.4 depend on numpy < 2.0.0

### DISCUSSION THREAD

**Discussion by Sciumo** [vllm-project/vllm#11176] [General]
pandas 2.2.3 depends on numpy 2.1.3

**Comment by JBurtn** [comment]
gguf had an issue with a numpy function, newbyteorder.
The PR to fix was just uploaded 9 hours ago (12/13/24). 
So long as you: 
   - use the most recent version of vllm.
   - install gguf from source.
 you should be fine.
